### PR TITLE
feature: crud with vue-router

### PR DESCRIPTION
**Pull request by locbq**

### CODE REVIEW

**Comment by locbq** [comment]
`vue-router` missing from dependency list
image

File: src/App.vue [hunk 1]
```diff
@@ -1,28 +1,56 @@
...
+    <div class="col-12">
+        <div id="nav">
+          <router-link :to="{ name: 'Todo List' }" class="btn btn-info">
+            Home
+          </router-link>
+           |
+          <router-link :to="{ name: 'Add Todo' }" class="btn btn-info">
+            Add Todo
+          </router-link>
+        </div>
```
Comment (locbq) [inline src/App.vue:12]: Move nav to a separate component

Comment (romanreign) [reply src/App.vue]: Fixed!

**Comment by romanreign** [comment]
> `vue-router` missing from dependency list
> image

added to fix!

**Comment by locbq** [comment]
you are so good please teach me master

**Pull request merged**